scripts/create_gw_molecule_database.py

```diff
@@ -143,19 +156,20 @@ def write_geom_database(geom_info):
 
 @dataclass
 class GeomInfo():
-    output_geom_database_filepath: str = 'gw5000.txt'
-    mol_info_filepath: str = '/Users/vandanrevanur/personal/codes/chemistry/goDatabase/data/gw5000/df_5k.json'
+    number_of_molecules : int = 62000
```

Comment:
The number of molecules is not 62,000. In the paper they rounded up from approximately 61,400.

Reply:
In the paper, it was called OE62, not GW62000.

However, I think it's better for the file path part to use something like "name_of_database = OE62" or "name_of_database = GW5000" rather than "GW+number_of_molecules". 

There might be a database called "pharmaceutical_molecules", and in that case you would not be entering the number of molecules to get the file path.

Reply:
Modified code accordingly.